jenkins/param-fb.yml

```diff
@@ -0,0 +1,121 @@
+- job:
+    name: percona-server-8.0-fb-param
```

Comment:
This job has nothing in common with Percona Server. It should be `fb-mysql-server-8.0` or more generic `mysql-server-8.0` (for testing both MySQL and FB-MySQL).

Reply:
ok, I see your point. I have fixed it.